cmake/CeleritasLibraryUtils.cmake

```diff
@@ -112,20 +122,28 @@ forward directly to CMake or alternatively to CudaRdc.
 
   See ``target_compile_options`` for additional detail.
 
-.. command:: celeritas_configure_file
 
-  Configure to the build "include" directory for later installation::
+Helper functions
+^^^^^^^^^^^^^^^^
 
-    celeritas_configure_file(<input> <output> [ARGS...])
+.. command:: celeritas_get_cuda_source_args
 
-  The ``<input>`` must be a relative path to the current source directory, and
-  the ``<output>` path is configured to the project build "include" directory.
+  Get a list of all source files in the argument that are CUDA language.
 
-.. command:: celeritas_polysource_append
+    celeritas_get_cuda_source_args(<var> [<source> ...])
 
-  Add C++ and CUDA/HIP source files based on the enabled options.
+.. command:: celeritas_add_library
```

Comment:
odd placement.  Wouldn't be better next to `celeritas_add_interface_library`?

Reply:
I thought this function was only used internal to this file but looks like it's used elsewhere. I can move it around.